example/docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: "3.9"
+services:
+  morp:
+    image: ghcr.io/jaconi-io/morp:0.2.3
```

Comment:
Should we reference `latest` here to make sure the latest image is being pulled?

Reply:
We don't publish a latest image (yet). This gets updated automatically, though. See my addition to `.releaserc`.